bridge-token-factory/src/lib.rs

```diff
@@ -107,6 +160,16 @@ pub struct BridgeTokenFactory {
     /// Mask determining all paused functions
     #[deprecated]
     paused: Mask,
+    /// Fee amount bound for deposit.
+    pub deposit_fee_bound: UnorderedMap<String, DepositTokenBounds>,
+    /// Fee amount bound for withdraw
+    pub withdraw_fee_bound: UnorderedMap<String, WithdrawTokenBounds>,
+    /// Fee percentage of each token for deposit
+    pub deposit_fee_percentage: UnorderedMap<String, DepositFeePercentage>,
+    /// Fee percentage of each token for withdraw
+    pub withdraw_fee_percentage: UnorderedMap<String, WithdrawFeePercentage>,
+    /// Withdraw fee percentage for each different silos for different tokens
+    pub withdraw_fee_percentage_per_silo: UnorderedMap<AccountId, SiloTokenFeeAmount>,
```

Comment:
You need to implement a migration since new fields were added.
You can also remove the `#[deprecated] paused: Mask` field, it was marked as `deprecated` just to avoid migration.